### PR TITLE
test: update test_rados_tool.sh, use POOL and OBJ var

### PR DESCRIPTION
**Pull request by Liuchang0812**
* replace hard-code pool name with $POOL
* replace hard-code object name with $OBJ
* introduce a new variable called $POOL_EC
* clean up pool
* simplify test case

ref to #12674 
@tchaikov @liewegas 

Signed-off-by: liuchang0812 <liuchang0812@gmail.com>

### CODE REVIEW

File: qa/workunits/rados/test_rados_tool.sh [hunk 1]
```diff
@@ -591,5 +567,10 @@ test_cleanup
 test_append
 test_put
 
+# clean up environment, delete pool
+$CEPH_TOOL osd pool delete $POOL $POOL --yes-i-really-really-mean-it
+$CEPH_TOOL osd pool delete $POOL_EC $POOL_EC --yes-i-really-really-mean-it
+$CEPH_TOOL osd pool delete $POOL_CP_TARGE $POOL_CP_TARGET --yes-i-really-really-mean-it
```
Comment (tchaikov) [inline qa/workunits/rados/test_rados_tool.sh]: s/POOL_CP_TARGE/POOL_CP_TARGET/

Comment (Liuchang0812) [inline qa/workunits/rados/test_rados_tool.sh]: done, thanks.

by the way, it seems that "ceph osd pool delete" will always returns 0 whether it's successfully or not.

Comment (Liuchang0812) [inline qa/workunits/rados/test_rados_tool.sh]: ok, i  have tried it.
ceph returns 0 if pool isn't exist.
ceph returns 1 if pool1 != pool2 and pool1 exists.

File: qa/workunits/rados/test_rados_tool.sh [hunk 2]
```diff
@@ -455,38 +460,24 @@ test_cleanup() {
...
   dd if=/dev/zero of=./rados_append_4k bs=4k count=1
-  $RADOS_TOOL -p rados_append append rados_append_obj ./rados_append_4k
-  $RADOS_TOOL -p rados_append get rados_append_obj ./rados_append_4k_out
-  orig_size=`ls -l ./rados_append_4k  | awk -F ' '  '{print $5}'`
-  read_size=`ls -l ./rados_append_4k_out | awk -F ' '  '{print $5}'`
-  if [ $orig_size -ne $read_size ];
-  then
-    die "Append failed expecting $orig_size read $read_size"
-  fi
+  $RADOS_TOOL -p $POOL append $OBJ ./rados_append_4k
```
Comment (tchaikov) [inline qa/workunits/rados/test_rados_tool.sh]: this changes the behavior of this test. prior to this change, we

1. create an empty object
1. append 4k "0" blob to it
1. compare it with the 4k blob

after this change

1. append 4k "0" blob to a non-exisitent object
1. compare it with the 4k blob

Comment (Liuchang0812) [inline qa/workunits/rados/test_rados_tool.sh]: done. very thanks for your review

**Comment by Liuchang0812** [comment]
Test this please

File: qa/workunits/rados/test_rados_tool.sh [hunk 3]
```diff
@@ -267,7 +271,7 @@ done
 run_expect_nosignal "$RADOS_TOOL" --pool "$POOL" bench 1 rand
 run_expect_nosignal "$RADOS_TOOL" --pool "$POOL" bench 1 seq
 
-set -e
+set -e
```
Comment (tchaikov) [inline qa/workunits/rados/test_rados_tool.sh]: trailing space.

Comment (Liuchang0812) [inline qa/workunits/rados/test_rados_tool.sh]: working on this

Comment (Liuchang0812) [inline qa/workunits/rados/test_rados_tool.sh]: done, I have squashed commits. thanks tchaikov

**Comment by Liuchang0812** [comment]
@tchaikov ping

**Pull request merged**